.github/ISSUE_TEMPLATE/i18n.md

```diff
@@ -1,36 +1,41 @@
 ---
 name: 🌐 Translating a new language?
 about: Start a new translation effort in your language
-title: '[i18n-<languageCode>] Translating docs to <languageName>'
+title: '🌐 [i18n-LANGCODE] Translating docs to LANGUAGE'
```

Comment:
Please keep the original `<>` and original names. You can switch to camel-cased names if you want but all caps name are more confusing and ut's not obvious at all they have to be replaced whereas something else (you should expect users will not read the note below 😅 )